oeps/oep-0001.rst

```diff
@@ -0,0 +1,318 @@
+
+An OEP can be assigned the status "Deferred". The OEP author or Arbiter can
+assign the OEP this status when no progress is being made on the OEP. Once a
+OEP is deferred, the OEP Author can re-assign it to draft status.
+
+An OEP can also be "Rejected" by the Arbiter. Perhaps after all is said and
+done it was not a good idea. It is still important to have a record of this
+fact. The "Withdrawn" status is similar - it means that the OEP author
+themselves has decided that the OEP is actually a bad idea, or has accepted
+that a competing proposal is a better alternative.
+
+When a OEP is Accepted, Rejected or Withdrawn, the OEP should be updated
+accordingly. In addition to updating the status field, at the very least the
+Resolution header should be added with a link to the relevant post in the edx-code
+mailing list archive or to the appropriate section of the PR.
+
+OEPs can also be superseded by a different OEP, rendering the original
+obsolete.
+
+The possible paths of the status of OEPs are as follows:
+
+.. image:: oep-0001/state-flow.png
+
+Please note that OEP statuses do not necessary coincide with the status of
+the pull request that contains the OEP. For example, OEPs that have been "Rejected"
+should still be merged, but should be marked as Rejected. This preserves the rationale
+and description of the OEP in the generated documentation.
+
+Likewise, an OEP that is in "Draft" may be merged to capture a set of edits,
+and to make the proposal more visible to community comment. From that point, additional
+pull requests can be opened to edit the "Draft" OEP, until it converges to being
+either "Accepted" or "Rejected".
+
+OEP Roles
+---------
+
+Each OEP will have an Arbiter (as described in `OEP Review & Resolution`_). The
+Arbiter will be chosen by Eddie Fagin (edX's Chief Archictect). The Arbiter
+will be the person making the final decision on whether the OEP should be Accepted,
+and as such, they should be knowledgable about the contents of the proposal, and
+willing to listen to arguments both for and against it by the rest of the community.
+The Author of the OEP will never be selected as the Arbiter of that OEP.
+
+The Arbiter is also responsible for helping the Author to elicit feedback from
+the community on the OEP, and for helping to move the OEP towards a final decision
+(whether that decision is Accepted, Rejected or Deferred). The Arbiter (in
+discussion with the Author) can merge an in-progress OEP (if it has reached a
+stage of relative stability) to allow for additional incremental updates.
+
+Finally, the Arbiter is responsible for the decision to transfer an OEP if the
+original Author has become unresponsive (as described in `Transferring OEP Ownership`_).
+
+
+OEP Maintenance
+---------------
+
+In general, OEPs are no longer modified after they have reached the Final state.
+They can be replaced by subsequent OEPs, however (OEPs that have been replaced
+are given the status "Replaced").
```

Comment:
I'm also assuming there is a class of edits that we are OK with making?

Examples:
* Formatting changes
* Grammatical and spelling fixes
* If Eddie were to leave us (hopefully never) we would want to change the name of the person who chooses Arbiters
* Additional links to other documents (OEPs, external resources etc)
* Additional diagrams and other non-substantive but clarifying material
* Descriptions of "as built" functionality and/or gap analysis between where we are and where we want to be

It's not clear to me if edits like those should cause the OEP to be replaced or not.